source/documentation/using_the_api/api_reference.md

```diff
@@ -1,8 +1,6 @@
 ## <a name="apireference"></a>API reference
 
-### Find out more about a register
-
-Path: GET /register
+### GET /register
```

Comment:
Is it possible to make these headers monospaced? E.g. `GET /register`. Helps when discriminating between prose and code.

Reply:
Yes, I can do that, but they won't appear monospaced in the ToC.